crates/stc_ts_file_analyzer/tests/pass/types/unknown/controlFlow/1.swc-stderr

```diff
@@ -0,0 +1,128 @@
+
+  x Type
+   ,-[$DIR/tests/pass/types/unknown/controlFlow/1.ts:4:3]
+ 4 | if (value === undefined) {
+   :     ^^^^^
+   `----
+
+Error: 
+  > T
+
+  x Type
+   ,-[$DIR/tests/pass/types/unknown/controlFlow/1.ts:4:3]
+ 4 | if (value === undefined) {
+   :               ^^^^^^^^^
+   `----
+
+Error: 
+  > undefined
+
+  x Type
+   ,-[$DIR/tests/pass/types/unknown/controlFlow/1.ts:4:3]
+ 4 | if (value === undefined) {
+   :     ^^^^^^^^^^^^^^^^^^^
+   `----
+
+Error: 
+  > boolean
+
+  x Type
+   ,-[$DIR/tests/pass/types/unknown/controlFlow/1.ts:5:5]
+ 5 | return value; // value: T extends unknown
+   :        ^^^^^
+   `----
+
+Error: 
+  > T
+
+  x Type
+   ,-[$DIR/tests/pass/types/unknown/controlFlow/1.ts:7:3]
+ 7 | value; // T & ({} | null)
+   : ^^^^^
+   `----
+
+Error: 
+  > (T & (null | {
+  | }))
+
+  x Type
+   ,-[$DIR/tests/pass/types/unknown/controlFlow/1.ts:8:3]
+ 8 | if (value === 42) {
+   :     ^^^^^
+   `----
+
+Error: 
+  > never
```

Comment:
This is wrong

Reply:
yes, this is wrong.

but, There is a reason why this value is never.
This is because find_var gets this value as undefined.
I don't know why I can't find the value right above.